src/test/java/teammates/test/cases/browsertests/InstructorFeedbackPageUiTest.java

```diff
@@ -721,80 +721,80 @@ public void testDatePickerScripts() {
         Calendar initialCal = Calendar.getInstance(TimeZone.getTimeZone("UTC"));
         initialCal.setTime(cal.getTime());
```

Comment:
It seems like `initialCal` here does not have any use anymore. See if `cal` (few lines above this) can be made into `initialCal` instead.